### PR TITLE
Fix documentation errors for the distributions: normal, beta, and chi-square

### PR DESCRIPTION
**Pull request by talgalili**
Detailed changes:
- removed trailing space
- chi_squared_cdf: fixed the wording.
- inverse_normal_cdf  - added the type.
- normal_cdf - changed from v to value (since that's the actual parameter in the code), and added the type.
- inverse_beta_cdf  - added the type.
- beta_cdf - changed from v to value (since that's the actual parameter in the code), and added the type.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by talgalili** [comment]
cc: @rongrong

**Comment by talgalili** [comment]
Hey @rongrong - any chance to merge this diff?
It's relatively minor, and fixes various small documentation issues we've already identified in previous review cycles.

Thanks!

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Mathematical Functions
 .. function:: chi_squared_cdf(df, value) -> double
 
     Compute the Chi-square cdf with given df (degrees of freedom) parameter:  P(N < value; df).
-    The df parameter must be positive real numbers and value must be a real value.
+    The df parameter must be a positive real number (of type DOUBLE) and value must be a real value.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/math.rst:46]: Value is also a DOUBLE right?

Comment (talgalili) [reply presto-docs/src/main/sphinx/functions/math.rst:51]: Thanks, done.

**Comment by talgalili** [comment]
@tdcmeehan thanks for the feedback. I've updated this PR with your suggestion.

Also, I took the opportunity to also author this small diff (stacked over this diff): #16696
It resolved another, long standing, issue with the documentation of probability distributions (namely, that they are not organized in alphabetical order, as is the convention in the rest of the docs). If you could please also review/accept that diff - it would be lovely.

With regards,
Tal

File: presto-docs/src/main/sphinx/functions/math.rst (hunk 1 above)
Comment (tdcmeehan) [reply presto-docs/src/main/sphinx/functions/math.rst:51]: Hmm, I still see that the type isn't mentioned for `value`.  Isn't this also significant?

Comment (talgalili) [reply presto-docs/src/main/sphinx/functions/math.rst:51]: Oops, I thought you meant the value of df is DOUBLE (which was already implemented). Yes, the type of 'value' is also DOUBLE. I'll fix it now.

Comment (talgalili) [reply presto-docs/src/main/sphinx/functions/math.rst]: NOW it's fixed.

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**